django-backend/fecfiler/contacts/migrations/0001_initial.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 3.2.11 on 2022-02-18 21:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = []
+
+    operations = [
+        migrations.CreateModel(
+            name="Contact",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("deleted", models.DateTimeField(blank=True, null=True)),
+                (
+                    "type",
+                    models.CharField(
+                        choices=[
+                            ("Committee", "Committee"),
+                            ("Organization", "Organization"),
+                            ("Individual", "Individual"),
+                            ("Candidate", "Candidate"),
+                        ],
+                        max_length=255,
+                    ),
+                ),
+                ("candidate_id", models.TextField(blank=True, null=True)),
+                ("committee_id", models.TextField(blank=True, null=True)),
+                ("name", models.TextField(blank=True, null=True)),
+                ("last_name", models.TextField(blank=True, null=True)),
+                ("first_name", models.TextField(blank=True, null=True)),
+                ("middle_name", models.TextField(blank=True, null=True)),
+                ("prefix", models.TextField(blank=True, null=True)),
+                ("suffix", models.TextField(blank=True, null=True)),
+                ("street_1", models.TextField()),
```

Comment:
Looking at all 4 contact specs, the fields street_1, city, state, zip, and country are required for all of them. Let's remove the null=True for those fields in the model so that nulls can't be entered.